pkg/proxy/backend/authenticator.go

```diff
@@ -220,6 +223,12 @@ func (auth *Authenticator) handshakeFirstTime(logger *zap.Logger, cctx ConnConte
 		case mysql.ErrHeader:
 			return pnet.ParseErrorPacket(serverPkt)
 		default: // mysql.AuthSwitchRequest, ShaCommand
+			if serverPkt[0] == mysql.AuthSwitchRequest {
+				pluginName = string(serverPkt[1 : bytes.IndexByte(serverPkt[1:], 0)+1])
+			} else if serverPkt[0] == 1 && pluginName == "caching_sha2_password" && len(serverPkt) == 2 && serverPkt[1] == 3 {
+				// skip caching_sha2_password fast path
```

Comment:
I think the fast path is not skipped. Why do you comment "skip"?

Reply:
You can comment if you don't like. For me, it 's 'skip fast path (packets)`.

Reply:
Just comment "caching_sha2_password fast path".